client/reader/site-subscriptions-manager/site-subscriptions-manager.tsx

```diff
@@ -1,6 +1,8 @@
 import {
 	__experimentalHStack as HStack,
 	__experimentalVStack as VStack,
+	// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+	__experimentalSpacer as Spacer,
```

Comment:
I wonder why the two imported values above (`HStack` and `VStack`) don't need the eslint comment as well. 🤔

Reply:
Oh, does this @mashikag's https://github.com/Automattic/wp-calypso/pull/79289#issuecomment-1632551804 explain that? 🙂

Reply:
Yes, I'll push an update :)